Headers/DebugServer2/Architecture/ARM/Branching.h

```diff
@@ -89,22 +89,22 @@ enum BranchDisp {
 };
 
 enum BranchCond {
-  kCondEQ,
-  kCondNE,
-  kCondCS,
-  kCondCC,
-  kCondMI,
-  kCondPL,
-  kCondVS,
-  kCondVC,
-  kCondHI,
-  kCondLS,
-  kCondGE,
-  kCondLT,
-  kCondLE,
-  kCondGT,
-  kCondAL,
-  kCondNV
+  kCondEQ = 0b0000,
+  kCondNE = 0b0001,
+  kCondCS = 0b0010,
+  kCondCC = 0b0011,
+  kCondMI = 0b0100,
+  kCondPL = 0b0101,
+  kCondVS = 0b0110,
+  kCondVC = 0b0111,
+  kCondHI = 0b1000,
+  kCondLS = 0b1001,
+  kCondGE = 0b1010,
+  kCondLT = 0b1011,
+  kCondGT = 0b1100,
+  kCondLE = 0b1101,
+  kCondAL = 0b1110,
+  kCondNV = 0b1111
 };
```

Comment:
Well, I know they aren't necessary from a functionality perspective, but I also suspect that the original author wouldn't have made the mistake they made if they had explicitly labeled the values, since the explicit values make it easier to check for correctness. On the other hand, we now know they're correct, so I guess it's fine to remove them.